draft-irtf-cfrg-sigma-protocols.md

```diff
@@ -88,7 +88,9 @@ informative:
 
 --- abstract
 
-This document describes interactive Sigma Protocols, a class of secure, general-purpose zero-knowledge proofs of knowledge consisting of three moves: commitment, challenge, and response. Concretely, the protocol allows one to prove knowledge of a secret witness without revealing any information about it.
+A Sigma Protocol is an interactive zero-knowledge proof of knowledge that allows a prover to convince a verifier of the validity of a statement. By the zero-knowledge property, no information beyond the truth of the statement is revealed.
+
+This document describes Sigma Protocols for proving knowledge of pre-images of linear maps in prime-order elliptic-curve groups. Examples include zero-knowledge proofs for discrete logarithm relations, ElGamal encryptions, Pedersen commitments, and range proofs.
```

Comment:
I appreciate the call-out that we are only defining linear maps in prime-order EC groups. Maybe explicitly calling that out in the abstract will address feedback about small-subgroup attacks, since they will be explicitly out of scope? (eg closes https://github.com/mmaker/draft-irtf-cfrg-sigma-protocols/issues/85)

Reply:
Make an additional not about prime order-ness in the group definitions section